google-cloud-clients/google-cloud-datastore/src/test/java/com/google/cloud/datastore/EntityTest.java

```diff
@@ -55,4 +55,9 @@ public void testCopyFromIncompleteEntity() throws Exception {
     assertNotEquals(INCOMPLETE_ENTITY, entity);
     assertEquals(INCOMPLETE_ENTITY.getProperties(), entity.getProperties());
   }
+
+  @Test
+  public void testEntitySize() throws Exception {
```

Comment:
Does this really throw exception?

Reply:
No,This method does not throw exception.